composer.json

```
@@ -18,7 +18,7 @@
 		"test:unit": "phpunit -c tests/phpunit.xml"
 	},
 	"require-dev": {
-		"nextcloud/coding-standard": "^0.3.0",
+		"nextcloud/coding-standard": "^1.0",
 		"psalm/phar": "^4.10",
 		"christophwurst/nextcloud": "dev-master@dev",
```

Should be replace by nextcloud/ocp

---

Also I would recommend to go with:
```suggestion
		"nextcloud/ocp": "dev-stable22",
```

Since that is the latest version you are trying to support.